synthesizer/src/program/mod.rs

```diff
@@ -34,7 +34,43 @@ mod parse;
 mod serialize;
 
 use console::{
-    network::prelude::*,
+    network::prelude::{
```

Comment:
Out of curiosity, how did you get this to unroll the exact objects? Or did you manually unroll it?